restapi/configure_cludod.go

```diff
@@ -92,6 +92,25 @@ func configureAPI(api *operations.CludodAPI) http.Handler {
 					return &principalID, nil
 				}
 			}
+		} else {
+			authz, err := conf.Server.NewGithubAuthorizer()
+			if err != nil {
+				api.Logger("ERROR: Failed to create authorizer: %v", err)
+				return nil, errors.New(500, "Failed to create authorizer: %v", err)
```

Comment:
"Failed to create authorizer" isn't a great logging message for operators to comb through.  Can we make it more applicable to the work being done instead of the implementation?  Something like "Failed to check public keys against github" or similar?

Reply:
Logging has been cleaned up a bit all around, but it does indeed still need some attention overall.